listings/ch13-functional-features/listing-13-05/output.txt

```diff
@@ -2,5 +2,5 @@ $ cargo run
    Compiling closure-example v0.1.0 (file:///projects/closure-example)
     Finished dev [unoptimized + debuginfo] target(s) in 0.43s
      Running `target/debug/closure-example`
-呼叫閉包前[1, 2, 3]
+呼叫閉包前 [1, 2, 3]
```

Comment:
Should either update the corresponding code example. And I also prefer a full-width colon instead of a space.